src/database/migrations/20230311231119-create-user.js

```diff
@@ -16,41 +16,34 @@ module.exports = {
         type: Sequelize.STRING
       },
       email: {
-        type: Sequelize.STRING,
-        allowNull: false,
+        type: Sequelize.STRING
       },
       password: {
-        type: Sequelize.STRING,
-        allowNull: false,
+        type: Sequelize.STRING
       },
-      emailVerificationToken: {
+      resetPasswordToken: {
         type: Sequelize.STRING,
-        allowNull: true,
-        defaultValue: ''
+        allowNull: true
       },
```

Comment:
Hope this doesn't cause any task to fail, because signup used to store token for future use (in account verification). and you replaced